### PR TITLE
Add /logout endpoint and tests

### PR DESCRIPTION
**Pull request by Trinovantes**

### CODE REVIEW

File: src-backend/registration/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+
+
+class ViewTest(TestCase):
```
Comment (Kapin) [inline src-backend/registration/tests/test_views.py]: can this have a better name

**Comment by Ommy** [comment]
+1 after rename of test

File: src-backend/registration/views.py [hunk 2]
```diff
@@ -65,3 +92,22 @@ def login(request):
...
+# ------------------------------------------------------------------------------
+
+@api_view(['POST'])
+@authentication_classes((TokenAuthentication,))
+@permission_classes((IsAuthenticated,))
+def logout(request):
+    success = False
+
+    if request.user:
+        Token.objects.get(user=request.user).delete()
```
Comment (ConnorCimowsky) [inline src-backend/registration/views.py]:
```python
token = Token.objects.get(user=request.user)
if token:
    token.delete()
```

**Comment by ConnorCimowsky** [comment]
:sparkles: LGTM after comments :sparkles:

**Pull request merged**